.changeset/light-mice-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@onflow/react-sdk": minor
+---
+
+Added useFlowAuthz hook
```

Comment:
Can this be more descriptive? It bubbles up tot release notes right?

Reply:
Implemented 👍